docs/preview/features/secret-store/provider/key-vault.md

```diff
@@ -48,6 +48,10 @@ public class Program
                         // Assing a configurable cached variant of the Azure Key Vault provider.
                         var cacheConfiguration = new CacheConfiguration(TimeSpan.FromMinutes(1));
                         builder.AddAzureKeyVaultWithManagedServiceIdentity(keyVaultUri, cacheConfiguration);
+
+                        // Adding the Azure Key Vault secret provider, using `.` instead of `:` when looking up secrets.
+                        // Example - When looking up `ServicePrincipal:ClientKey` it will be changed to `ServicePrincipal.ClientKey`.
```

Comment:
[Using `.` is not allowed](https://docs.microsoft.com/en-us/azure/key-vault/general/about-keys-secrets-certificates#:~:text=Key%20Vault%20name%20must%20be,a%2Dz%2C%20A%2DZ%2C%20and%20%2D.&text=The%20type%20of%20the%20object,%22%2C%20or%20'certificates'.&text=An%20object%2Dname%20is%20a,unique%20within%20a%20Key%20Vault.) and one of the reasons I was asking for this 😎 

```suggestion
                        // Example - When looking up `ServicePrincipal:ClientKey` it will be changed to `ServicePrincipal-ClientKey`.
```